tasks/main.yml

```diff
@@ -11,10 +11,12 @@
     - restart omero-web
 
 - name: omero web apps | add application web configuration
-  become: yes
+  become: true
   template:
-    dest: "{{ omero_web_apps_omeroweb_config_dir }}/{{ omero_web_apps_config_name }}.omero"
-    force: yes
+    dest: >-
+      {{ omero_web_apps_omeroweb_config_dir }}/
+      {{ omero_web_apps_config_name }}.omero
```

Comment:
This is rendered with an extra space: `{{ omero_web_apps_omeroweb_config_dir }}/ {{ omero_web_apps_config_name }}.omero`

Reply:
I think d2933fd is the correct way to split multi-lines without spaces in YML.